src/docker/composeappengine.cc

```diff
@@ -291,37 +332,32 @@ void ComposeAppEngine::installApp(const App& app) {
 
 void ComposeAppEngine::start(const App& app) {
   LOG_INFO << "Starting App: " << app.name << " -> " << app.uri;
-  exec(compose_ + "up --remove-orphans -d", "failed to start App", boost::process::start_dir = appRoot(app));
+  exec(compose_ + "up --remove-orphans -d", "failed to bring Compose App up", boost::process::start_dir = appRoot(app));
 }
 
 bool ComposeAppEngine::areContainersCreated(const App& app) {
-  bool result{false};
-  try {
-    ComposeInfo info((appRoot(app) / ComposeFile).string());
-    std::vector<Json::Value> services = info.getServices();
-    if (services.empty()) {
-      LOG_ERROR << "App: " << app.name << ", no services in docker file!";
-      return false;
-    }
+  bool result{true};
+  ComposeInfo info((appRoot(app) / ComposeFile).string());
+  std::vector<Json::Value> services = info.getServices();
+  if (services.empty()) {
+    throw std::runtime_error("No services found in App's compose file");
```

Comment:
this is dropping the App's name. I assume the higher level exception handler was already printing the app's name?

Reply:
yep, this is more concise approach, an error message in output in one place, not even in the exception handler, in liteclient.